docs/sql-ref-syntax-aux-describe-function.md

```diff
@@ -18,5 +18,40 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+Return the metadata such as qualified function name, implementing class and usage of an existing function.
+Input parameter function name may be optionally qualified. If the function is not qualified then the function is
+resolved from the current database. If EXTENDED option is specified then additionally extended usage information
```

Comment:
Back-tick `EXTENDED`, and add "the" before.

Reply:
@srowen Changed. Sean, i also backticked the function_name. Is that ok ?